src/globalStyles.ts

```diff
@@ -114,6 +114,8 @@ export const globalStyles = `
     --callbacks-title-bg: #F5F5F5;
     --callbacks-title-outline: #C2C2C2;
     --callbacks-details-bg: #FAFAFA;
+
+    --table: ${palette.white};
```

Comment:
Remove both `--table` vars

Reply:
Could you explain further why I should do that? Im using the `--table` vars in the `themes.ts` file so that the background of the table is the correct colour.

Reply:
sorry meant to get rid of that , fixed !